website/docs/tutorials/react-native.md

```diff
@@ -128,21 +130,21 @@ The interplay of `I18nProvider` and `useLingui` is shown in the following simpli
 
 ```tsx
 import { I18nProvider } from "@lingui/react";
-import { t, Trans } from "@lingui/macro";
+import { Trans, useLingui } from "@lingui/macro";
```

Comment:
The sentence above looks like not relevant anymore

```
If we cannot use `Trans`, we can use the `useLingui` hook to get hold of the `i18n` instance ourselves and get the translations from there.
```

at least, there we talk about hook, but use the macro in the example

Reply:
Agree this paragraph should be rephrased. Could you help with it? The idea, i think, if we are not in JSX we could use `const { t } = useLingui()` macro

Reply:
Will check after merging